content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'How to use the Prisma Postgres Partner API'
+metaTitle: 'How to use the Prisma Postgres Partner API'
+description: 'Learn how to use the Prisma Postgres Partner API'
+sidebar_label: 'Partner API'
+image: '/img/guides/prisma-partner-api-cover.png'
+community_section: true
+---
+
+## Introduction
+
+This guide walks you through how to use the Prisma Postgres Partner API, which powers the [`npx create-db`](https://create-db.prisma.io/) command.
+
+You'll learn how a Prisma Postgres database is provisioned, how you can transfer it to a specific workspace, and how the process is secured using OAuth2. By the end, you'll understand the full flow and how to leverage it in your own projects.
+
+This guide references the actual implementation in the `npx create-db` CLI and Cloudflare Workers for a real world example. The repo for the `npx create-db` is [here](https://github.com/prisma/create-db) which can be used as a reference for how to use the Partner API in your own projects.
+
+## Overview
```

Comment:
Can we add a paragraph and callouts to help partners understand how this would relate to their app? E.g. the 2 workers would be part of their existing backend, and the CLI shows a simplistic implementation of how they can make the experience their own in their own product.

So, basically, in addition to explaining the create-db side of things, also put it into the context of a (potential) partner who's reading this guide and help them make sense of these components and why/how they're relevant to them.